newIDE/app/src/InstancesEditor/index.js

```diff
@@ -39,6 +39,12 @@ const styles = {
   dropCursor: { cursor: 'copy' },
 };
 
+const isNoDialogOpened = (): boolean => {
```

Comment:
Do you think this could be an util? Somehow I have the feeling that having this function in its own file would help us, when we upgrade Material-UI one day, to remember/find "oh there is this query selector that is checking a specific class, let's check the name of the class is still up to date"

Reply:
It could a file `UI/MaterialUISpecificUtil.js` ?

Reply:
Perfect